src/java.base/share/classes/java/security/spec/PSSParameterSpec.java

```diff
@@ -64,13 +64,18 @@
  *   ...  -- Allows for future expansion --
  * }
  * </pre>
- * <p>Note: the PSSParameterSpec.DEFAULT uses the following:
+ * <p>Note: the {@code PSSParameterSpec.DEFAULT} uses the following:
```

Comment:
Suggest rewording as:
`Note: {@code PSSParameterSpec.DEFAULT} uses the following values:`

Reply:
This part has been removed to reduce duplication since there is already javadoc for DEFAULT below.